core/commands/repo.go

```diff
@@ -203,9 +215,14 @@ Version         string The repo version.
 					fmt.Fprintf(wtr, "StorageMax:\t%d\n", stat.StorageMax)
 				}
 			}
+
+			if sizeOnly {
+				return nil
+			}
+
+			fmt.Fprintf(wtr, "NumObjects:\t%d\n", stat.NumObjects)
```

Comment:
This is technically a breaking change but I doubt anyone is relying on this order.

Reply:
I hope not. I can turn it around though and leave NumObjects as first line.

Reply:
I'll just change it. It's marginally better to not change the "API".